### PR TITLE
systemd-analyze: Add new meson configure option for controlling systemd-analyze support

### PR DESCRIPTION
**Pull request by jsmolic**
Enables building systemd without systemd-analyze, which in
return saves approx. 4 MB of space upon installing systemd.
Also extends features list with new configuration options.

Signed-off-by: Jakov Smolic <jakov.smolic@sartura.hr>
Signed-off-by: Luka Perkov <luka.perkov@sartura.hr>

### CODE REVIEW

File: meson_options.txt [hunk 1]
```diff
@@ -364,3 +364,5 @@ option('llvm-fuzz', type : 'boolean', value : 'false',
        description : 'build against LLVM libFuzzer')
 option('kernel-install', type: 'boolean', value: 'true',
        description : 'include kernel-install feature')
```
Comment (keszybz) [inline meson_options.txt]: Maybe this should be "install kernel-install and associated files"

File: meson_options.txt [hunk 2]
```diff
@@ -364,3 +364,5 @@ option('llvm-fuzz', type : 'boolean', value : 'false',
...
+option('analyze', type: 'boolean', value: 'true',
+       description : 'include systemd-analyze')
```
Comment (keszybz) [inline meson_options.txt]: ... and this "install systemd-analyze".

"include" is very vague.

File: meson.build [hunk 3]
```diff
@@ -3500,6 +3500,8 @@ foreach tuple : [
         ['link-systemctl-shared', get_option('link-systemctl-shared')],
         ['link-networkd-shared',  get_option('link-networkd-shared')],
         ['link-timesyncd-shared', get_option('link-timesyncd-shared')],
+	['kernel-install'],        get_option('kernel-install')],
+	['systemd-analyze'],      get_option('analyze')],
```
Comment (keszybz) [inline meson.build]: Spaces please.

File: meson_options.txt (hunk 2 above)
Comment (jsmolic) [reply meson_options.txt]: I agree, I should have specified this a bit better.
Pushed to https://github.com/systemd/systemd/commit/f30bf4d937de4ef69fedc4e73fabe43b8955bed6, should be good now.

**Pull request merged**